frontend/src/components/grantaccess/form/ForestClientInput.vue

```diff
@@ -83,7 +81,7 @@ const verifyForestClientNumber = async (forestClientNumbers: string) => {
     forestClientNumbersInput.value = forestNumbers.toString();
 };
 
-const isForestClientNumberNotAdded = (forestClientNumber: string) => {
+const isForestClientNumberAdded = (forestClientNumber: string) => {
```

Comment:
Hi Jota, I think the logic feels weird... so if we rename it to "isForestClientNumberAdded", then I feel it should return true when found the record. But in this case, the code logic returns false when found. So I think if we rename it, we want to remove the "!" from the code, and it will return true when found. So on line 38, it should be 
`if (isForestClientNumberAdded(forestClientNumber))`, if forest client is added, we print error message. Currently it's confused.

Reply:
Changed the logic, sorry about that! I did not thought that would make hard to understand